_docker/environments/drupal-dev/rhd.settings.php

```diff
@@ -23,7 +23,7 @@
   'username' => 'drupal',
   'password' => 'drupal',
   'prefix' => '',
-  'host' => 'docker',
+  'host' => 'drupalmysql',
```

Comment:
I think if you commit this it will break running Drupal for people that are not using Docker on their local machine (which I think is the majority at the minute). I think the majority of people are using Drupal outside of Docker currently, so I'd rather leave it working for the majority and then try and get a fix for the Docker environment at a later date.

I have raised https://issues.jboss.org/browse/DEVELOPER-4183 to take a look at this.